CHANGE-NOTES.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## v3.7.2
+- Added support for IntelliJ 2023.1.
```

Comment:
Hmmm this support has actually been added already back in v3.4.0... there isn't anything about this particular change that adds support for 2023.1 (other than changing the build target to 2023.1, but that's of relatively lesser importance) 🤔